Source/WebCore/inspector/agents/InspectorCSSAgent.cpp

```diff
@@ -1044,6 +1045,19 @@ static bool hasJSEventListener(Node& node)
     return false;
 }
 
+static bool hasAssignedNodes(Node& node)
```

Comment:
Since "assigned nodes" is an obscure term...

Naming suggestion: `isSlotElementWithAssignedNodes`